README.md

```diff
@@ -161,102 +281,71 @@ tags.add("foo");
-
-As example for Amazon Kindle Fire it looks like this:
-
-```java
-Installation installation = new Installation("installation-id", NotificationPlatform.Adm, "adm-push-channel");
-hub.createOrUpdateInstallation(installation);
-```
-	
-Need to add something new - just do it:
-
-```java
-installation.addTag("foo");
-installation.addTemplate("template1", new InstallationTemplate("{\"data\":{\"key1\":\"$(value1)\"}}","tag-for-template1"));
-installation.addTemplate("template2", new InstallationTemplate("{\"data\":{\"key2\":\"$(value2)\"}}","tag-for-template2"));
-hub.createOrUpdateInstallation(installation);
-```
-
-For advanced scenarios we have partial update capability which allows to modify only particular properties of the installation object. Basically partial update is subset of [JSON Patch](https://tools.ietf.org/html/rfc6902/) operations you can run against Installation object.
-
-```java
-PartialUpdateOperation addChannel = new PartialUpdateOperation(UpdateOperationType.Add, "/pushChannel", "adm-push-channel2");
-PartialUpdateOperation addTag = new PartialUpdateOperation(UpdateOperationType.Add, "/tags", "bar");
-PartialUpdateOperation replaceTemplate = new PartialUpdateOperation(UpdateOperationType.Replace, "/templates/template1", new InstallationTemplate("{\"data\":{\"key3\":\"$(value3)\"}}","tag-for-template1")).toJson());
-hub.patchInstallation("installation-id", addChannel, addTag, replaceTemplate);
+NotificationOutcome outcome = hub.sendNotificationn);
 ```
 
-Delete Installation:
-
-```java
-hub.deleteInstallation(installation.getInstallationId());
-```
-	
-Keep in mind that CreateOrUpdate, Patch and Delete are eventually consistent with Get. In fact operation just goes to the system queue during the call and will be executed in background. Moreover Get is not designed for main runtime scenario but just for debug and troubleshooting purposes, it is tightly throttled by the service.
+### Send To An Installation ID:
 	
 Send flow for Installations is the same as for Registrations. We've just introduced an option to target notification to the particular Installation - just use tag "InstallationId:{desired-id}". For case above it would look like this:
 
 ```java
 Notification n = Notification.createWindowsNotification("WNS body");
-hub.sendNotification(n, "InstallationId:{installation-id}");
+NotificationOutcome outcome = hub.sendNotification(n, "InstallationId:{installation-id}");
```

Comment:
The tag is missing the leading `$`. Should be `$InstallationId:{installation-id}`.

Reply:
Fixed